### PR TITLE
Bumped version to 2.2.6-SNAPSHOT

### PR DESCRIPTION
**Pull request by alebastrov**

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -20,5 +20,7 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
+    - name: settings.xml
+      run: cat /home/runner/.m2/settings.xml
     - name: Build with Maven
-      run: mvn clean install --batch-mode  -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn --file pom.xml
+      run: mvn clean install -B -U -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn --file pom.xml
```
Comment (treaz) [inline .github/workflows/maven.yml]: clean is not needed, -U is also not needed.
"You can specify the runner type for each job in a workflow. Each job in a workflow executes in a fresh instance of the virtual machine. All steps in the job execute in the same instance of the virtual machine, allowing the actions in that job to share information using the filesystem."
from https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners

File: dxa-model-service-in-process/pom.xml [hunk 2]
```diff
@@ -5,38 +5,38 @@
...
-                                    <classifier>spring-boot</classifier>
-                                </configuration>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-    </profiles>
+<!--    <profiles>-->
```
Comment (treaz) [inline dxa-model-service-in-process/pom.xml]: remove entirely please

File: pom.xml [hunk 3]
```diff
@@ -56,18 +55,27 @@
...
         </repository>
-        <snapshotRepository>
-            <id>snapshots-nexus</id>
-            <name>SDL Snapshots</name>
-            <url>https://nexus.sdl.com/repository/snapshots</url>
-        </snapshotRepository>
-    </distributionManagement>
+    </repositories>
+
+<!--    <distributionManagement>-->
```
Comment (treaz) [inline pom.xml]: why did you remove this? we might need it for the release process, no?

File: .github/workflows/maven.yml (hunk 1 above)
Comment (NIkonDSL) [reply .github/workflows/maven.yml]: done in develop

File: pom.xml (hunk 3 above)
Comment (NIkonDSL) [reply pom.xml]: reverted back

File: dxa-model-service-in-process/pom.xml (hunk 2 above)
Comment (NIkonDSL) [reply dxa-model-service-in-process/pom.xml]: done
